### PR TITLE
Show far future as date+year when using timeago

### PR DESCRIPTION
**Pull request by e2**
- also fix tag tests by sorting attributes
- fix Rack setup for `rake test` task

### CODE REVIEW

File: app/assets/javascripts/local_time.js.coffee [hunk 1]
```diff
@@ -143,7 +143,9 @@ class RelativeTime
   relativeWeekday: ->
     daysPassed = @calendarDate.daysPassed()
 
-    if daysPassed > 6
+    if daysPassed < 0
+      null
+    else if daysPassed > 6
```
Comment (javan) [inline app/assets/javascripts/local_time.js.coffee]: How about expressing this whole method as a switch statement?

```coffee
  relativeWeekday: ->
    switch @calendarDate.daysPassed()
      when 0
        "today"
      when 1
        "yesterday"
      when 2,3,4,5,6
        strftime @date, "%A"
```

File: test/helpers/local_time_helper_test.rb [hunk 2]
```diff
@@ -17,6 +17,11 @@ class LocalTimeHelperTest < TestCase
   include ActionView::Helpers::DateHelper, ActionView::Helpers::TagHelper
   include LocalTimeHelper
 
+  def normalize(html)
+    n = Nokogiri::HTML(html).xpath('//time').first
```
Comment (javan) [inline test/helpers/local_time_helper_test.rb]: Need to `require 'nokogiri'` too.

File: app/assets/javascripts/local_time.js.coffee (hunk 1 above)
Comment (bloot) [inline app/assets/javascripts/local_time.js.coffee]: :+1:

File: test/helpers/local_time_helper_test.rb (hunk 2 above)
Comment (bloot) [inline test/helpers/local_time_helper_test.rb]: I'm not a big fan of using `n` or `x` etc. in variable naming. It makes stuff kind of unclear, especially in manipulation like you are doing later on ( where you are using `n` again ). I would recommend using something like `document` here and then getting a little bit more descriptive in `#map`'s vars.

Comment (e2) [inline test/helpers/local_time_helper_test.rb]: Thanks guys, I left it as a quick hack and then forgot about it while doing the other changes.

**Comment by e2** [comment]
The attribute sorting is only one tag level deep, but it would be overkill here otherwise.

I was considering rewriting the tests to check with selector assertions, but there seemed to be little value added (I'd do it if it was RSpec - but the current more "hackish" version fits the "minitest culture" better).

**Comment by e2** [comment]
Sorry for the rebasing - I didn't anticipate the feedback.

**Comment by e2** [comment]
I used `rails-dom-testing` instead of the normalize.

I'm done. Let me know if there's any other improvements, thanks!

**Comment by javan** [comment]
Thank you! I just published v1.0.2. :beers:

**Comment by e2** [comment]
Thanks for making such a nice gem!

**Pull request merged**